auth_saml/data/ir_config_parameter.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<odoo noupdate="1">
+
+    <!-- Add a setting; to be manipulated via "general settings". -->
+
+    <record id="allow_saml_uid_and_internal_password" model="ir.config_parameter">
+        <field name="key">auth_saml.allow_saml.uid_and_internal_password</field>
+        <field name="value">0</field>
```

Comment:
This file could be removed, as you can call `.get_param(name, 0)` and it will return `0` if missing.